### DISCUSSION TITLE
Unable to manage DNS records for my domain.

### DISCUSSION THREAD

**Discussion by mayanksingh2298** [vercel/community#5165] [Help]
### Summary

I have bought a domain on namecheap and set the vercel nameserver there.

Now when I added the domain on vercel for the first time it allowed me to set the various DNS records that I needed.

I want to add another DNS records now. I don't see the option where I can manage the DNS records.

### Steps to Reproduce

add a domain in certain project.

set the vercel nameserver where you bought the domain.

Try to open DNS management settings

**Comment by amyegan** [comment]
Hey @mayanksingh2298. Can you share screenshots of what you see in Project Settings or your Domains page? I want to be sure I'm looking in the right place to help you get this working.

**Reply by bk-info** [reply]
Sorry @amyegan, I meant my comment to be a reply to you. Please advise.

**Comment by bk-info** [comment]
Hi I am having the same issue. Here are screenshots of my Project Settings, when I click Edit for each domain, and when I click Advanced on the left-hand side. No where can I manage my DNS records. 
<img width="1272" alt="Screen Shot 2024-03-07 at 3 29 02 PM" src="https://github.com/vercel/community/assets/162641390/59c4f4e3-066d-43b3-ac76-20d743bab926">
<img width="1231" alt="Screen Shot 2024-03-07 at 3 30 38 PM" src="https://github.com/vercel/community/assets/162641390/02d9cc82-33fd-42d8-937a-092f510fd662">
<img width="1289" alt="Screen Shot 2024-03-07 at 3 31 15 PM" src="https://github.com/vercel/community/assets/162641390/f4ec0d08-a314-4003-a217-82b0de68c7c9">

**Comment by operacionesBlueCorner** [comment]
Me esta pasando lo mismo, pudieron encointrar alguna solcuión? ayer 28 de marzo me aparecia con normalidad, y de la nada hoy no aparece

**Comment by amyegan** [comment]
DNS records can be managed outside of project settings. You should be able to see your custom domains listed either on your [account setting domains page](https://vercel.com/account/domains) or your team's Domains tab.

<img width="355" alt="image" src="https://github.com/vercel/community/assets/8247105/66c24f7e-4d26-4062-8920-35a42656fec0">
